lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/MavenLemminxExtension.java

```diff
@@ -92,6 +104,7 @@
 	private BuildPluginManager buildPluginManager;
 
 	XMLMavenSettings settings = new XMLMavenSettings();
+	private URIResolverExtensionManager resolverExtensionManager = new URIResolverExtensionManager();

```

Comment:
Don't instantiate URIResolverExtensionManger but get it in start method with 

```java
public void start(InitializeParams params, XMLExtensionsRegistry registry) {
	this.resolverExtensionManager  = registry.getResolverExtensionManager();
...
```

Reply:
ok